docs/env_var_example_reference.md

```diff
@@ -42,8 +41,30 @@ Because a task has a name that would not be implicitly useful as the default val
       end
     end
 
+### A task author wants input from the user, but not to print the environment's value
```

Comment:
This might be more clear as something like: "A task author wants input from the user, but does not want to print the environment variable's value"